pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -117,7 +118,12 @@ stdenv.mkDerivation rec {
     ] ++ stdenv.lib.optional (stdenv.hostPlatform.libc == "musl") "NO_AFFINITY=1"
     ++ mapAttrsToList (var: val: var + "=" + val) config;
 
-  patches = []; # TODO: Remove on next mass-rebuild
+    patches = [
+      (fetchpatch {
+        url = "https://github.com/xianyi/OpenBLAS/pull/1667.patch";
```

Comment:
Doesn't the contents change once someone pushes an update to the PR? I know it's been merged, but still. 
The following should be stable:
https://github.com/xianyi/OpenBLAS/commit/5f2a3c05cd0e3872be3c5686b9da6b627658eeb7.patch

Reply:
Yes, but since its merged I considered that  safe. I'll change it though.

A little rant: Github didn't notify me about this review. Just swallowed the notification. I was just about to ask for a review when I saw it.

Reply:
I changed the link. The reason I initially chose the `/pull/1667.patch` format is that conveniently doubles as information about the source. But a comment is better for that anyways.